.github/workflows/beta-build.yml

```diff
@@ -60,7 +59,7 @@ jobs:
           NODE_AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
       # Push tag
-      - run: git push origin beta -f
+      - run: git push origin beta
```

Comment:
Wondering if the changes are related to no longer needing a forced push.
Either way, that's good!